week3/homework/src/index.js

```diff
@@ -1,3 +1,140 @@
 'use strict';
+const express = require('express');
+const app = express();
+const bodyParser = require("body-parser");
+const fs = require("fs");
+app.use(bodyParser.json());
 
-// TODO: Write the homework code in this file
+app.get('/list', function (request, response) {
+  fs.readFile('./toDo.json', 'utf8', function (error, data) {
+    if (error) {
+      console.log(error);
+      return;
+    }
+    const myToDoList = JSON.parse(data);
+    if (Object.values(myToDoList).length === 0) {
+      response.end('You have not any plan in your To Do List');
+      return;
+    }
+    response.json(myToDoList);
+  });
+});
+
+app.get('/list/:id', function (request, response) {
+  fs.readFile('./toDo.json', 'utf8', function (error, data) {
```

Comment:
This functionality is used in multiple places, you should move the logic to a function, which can be called from the different places.